src/evaluation/WorkspaceManager.jl

```diff
@@ -260,6 +256,15 @@ function eval_in_workspace(session_notebook::Union{Tuple{ServerSession,Notebook}
     nothing
 end
 
+function format_fetch_in_workspace(session_notebook::Union{Tuple{ServerSession,Notebook},Workspace}, cell_id, ends_with_semicolon, showmore_id::Union{PlutoRunner.ObjectID, Nothing}=nothing)
+    workspace = get_workspace(session_notebook)
+    
```

Comment:
eval_format_fetch_in_workspace changes directory before acting, it might be best if this function did the same: formatting code may also depend on the working directory.